keystores/src/test/java/com/palantir/remoting3/config/ssl/DropwizardSslClientAuthTests.java

```diff
@@ -52,6 +52,8 @@
                     TestConstants.SERVER_KEY_STORE_JKS_PASSWORD),
             ConfigOverride.config("server.applicationConnectors[0].trustStorePath",
                     TestConstants.CA_TRUST_STORE_PATH.toString()),
+            ConfigOverride.config("server.applicationConnectors[0].trustStorePassword",
```

Comment:
As of https://github.com/eclipse/jetty.project/commit/a8d08df84fb7b6944e3bd11adb724fd05c29e357 truststore password will default to keystore password which will fail upon reading the jks keystore. Internally we require client cert auth truststore to come with passwords

Reply:
ok